core/domain/rights_manager.py

```diff
@@ -628,6 +628,16 @@ def _publish_activity(committer_id, activity_id, activity_type):
         committer_id, activity_id, activity_type, ACTIVITY_STATUS_PUBLIC,
         '%s published.' % activity_type)
 
+    #Add contribution dates to the owners and editors
```

Comment:
Yes, I can't think of any reasons why contributing to a collection needs to be more specific than explorations. I think if a collection is edited in any way, then that would be a contribution datetime. Whether it's a notable contribution is perhaps a different discussion, but any concerns in that area also apply to explorations. I don't think collections need to be handled in any special manner compared to explorations.